metron-platform/metron-elasticsearch/pom.xml

```diff
@@ -156,6 +168,19 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <!-- must be near the top of the dependencies to force retrieval of the correct netty version -->
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
+            <version>4.1.13.Final</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.elasticsearch.plugin</groupId>
+            <artifactId>transport-netty4-client</artifactId>
+            <version>${global_elasticsearch_version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
While the compile time dependencies for Elasticsearch are walled-off in the `elasticsearch-shaded` project, the same is not done for the test dependencies, including the use of `ElasticsearchComponent`.  This is why we have this specific dependency on Elastic here.

Other changes in this PR had an unexpected ripple effect causing multiple, incompatible versions of Netty to get pulled in.  I had  to explicitly exclude that from some dependencies, then create an explicit dependency on the version of Netty we need.  

Order also matters here.  If this is too low in the dependency list, the integration tests will fail.